src/main/java/ru/hogwarts/school/controller/StudentController.java

```diff
@@ -0,0 +1,65 @@
+package ru.hogwarts.school.controller;
+
+import io.swagger.v3.oas.annotations.Operation;
+import org.springframework.web.bind.annotation.*;
+import ru.hogwarts.school.entity.Faculty;
+import ru.hogwarts.school.entity.Student;
+import ru.hogwarts.school.service.StudentService;
+
+import java.util.List;
+
+@RestController
+@RequestMapping("/student")
+public class StudentController {
+    private final StudentService studentService;
+
+    public StudentController(StudentService studentService) {
+        this.studentService = studentService;
+    }
+
+    @PostMapping
+    public Student postStudent(@RequestBody Student student) {
+        return studentService.postStudent(student);
+    }
+
+    @GetMapping
+    @Operation(summary = "Get all students")
+    public List<Student> getAllStudents() {
+        return studentService.findAll();
+    }
+
+
+    @GetMapping(value = "/age",params = "age")
+    @Operation(summary = "Get students by age")
+    public List<Student> getStudentsByAge(@RequestParam Integer age) {
+        return studentService.findByAge(age);
+    }
+
+    @GetMapping(value = "/age/between", params = {"lowerAge, upperAge"})
```

Comment:
```suggestion
    @GetMapping(params = {"lowerAge, upperAge"})
```
снова та же ошибка - в пути ничего писать не нужно здесь